player.js

```diff
@@ -0,0 +1,63 @@
+var Scrabble = require("./scrabble.js");
+var scrabble = new Scrabble();
+
+var Player = function(name) {
+  this.name =  name;
+  this.plays = [];
+};
+
+Player.prototype.play = function(word) {
+  if (this.hasWon()) {
+    return false
+  }
+  this.plays.push(word);
+}
+
+Player.prototype.wordScores = function() {
+  var scores = [];
+  for (word of this.plays) {
+    scores.push(scrabble.score(word));
+  }
+
```

Comment:
This very minor but you could probably remove the extra line here.  It only pops out to me b/c I'm constantly removing them from my code at work. :)  Check Javascript Lint(ers) http://www.javascriptlint.com/  These generally pick up common mistakes.